src/admin/queries.js

```diff
@@ -149,3 +149,47 @@ export const ALL_JOBS = gql`
     }
   }
 `;
+
+// Get all banners
+export const ALL_BANNERS = gql`
+  query Banners {
```

Comment:
I think we can filter this by `enabled: true` to reduce what's being sent over the network?

Reply:
Yeah, I was suggesting that you add a variable to the query to allow you to specify if you want enabled banners or not:
```
query Banners($enabled: Boolean) {
    ...
}
```